src/UI/Settings/SettingsLayout.js

```diff
@@ -240,6 +243,7 @@ module.exports = Marionette.Layout.extend({
     },
 
     _toggleAdvancedSettings : function() {
+        console.log("Advance Toggled")
```

Comment:
I would suggest moving this down after you have the value and log the value as well,

`console.log("Advance Toggled. ", checked)`